plugins/Morpheus/stylesheets/less/_variables.less

```diff
@@ -17,6 +17,8 @@
 @theme-color-background-highContrast:  #202020;
 @theme-color-base-series:              #ee3024;
 
+@theme-color-border: @color-silver-l80;
```

Comment:
> I was really confused by that variable and wouldn't know when to use it.

On the opposite that's why I created the variable: `@color-silver-l80` is very confusing as we don't know when to use it, it doesn't represent anything except that it's grey. A border color is used for borders (tables, h1, hr, list-group, … anything that has a default border, so obviously you exclude cases where the border color is special like for alerts).